src/mem/mmap/mmap_kernel.c

```diff
@@ -41,7 +41,6 @@ int mmap_kernel_init(void) {
 	emmap = task_resource_mmap(task_kernel_task());
 
 	dlist_foreach_entry(marea, &early_emmap.marea_list, mmap_link) {
-		dlist_del_init(&marea->mmap_link);
```

Comment:
This is required?

Reply:
This has been deleted by mistake. I'll fix.